bsi_zoo/metrics.py

```diff
@@ -152,7 +152,7 @@ def f1(x, x_hat, orientation_type, *args, **kwargs):
     return f1_score(active_set, active_set_hat)
 
 
-def nll(x, x_hat, *args, **kwargs):
+def nll(x, x_hat, orientation_type, *args, **kwargs):
```

Comment:
orientation_type needs to collected from kwargs

Reply:
It was written this way in other functions in metrics.py